src/features/contacts/edit/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect, useState } from "react";
```

Comment:
it's better use the same control for "add" and "edit", or inherit them from some basic control to not duplicate code

Reply:
Fixed.